src/features/berth/berthPage/berthLegend/applicationSelector/ApplicationSelectorContainer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { EXCHANGE_APPLICATION_LIMIT } from '../../../../../common/utils/constants';
+import { switchApplication as switchApplicationAction } from '../../../../../redux/actions/ApplicationActions';
+import {
+  resetBerthLimit as resetBerthLimitAction,
+  setBerthLimit as setBirthLimitAction,
+} from '../../../../../redux/actions/BerthActions';
+import { Store } from '../../../../../redux/types';
+import { ApplicationOptions } from '../../../../../common/types/applicationType';
+import './applicationSelector.scss';
+import ApplicationSelector from './ApplicationSelector';
+import useAutoDismissAlert from './useAutoDismissAlert';
+
+export type ApplicationSelectorContainerProps = {
+  berthsApplicationType: string;
+  className?: string;
+  resetBerthLimit: Function;
+  selectedBerthCount: number;
+  setBerthLimit: Function;
+  switchApplication: Function;
+};
+
+export const _ApplicationSelectorContainer = ({
```

Comment:
Why did you prefix it with an underscore?

Reply:
A named export shouldn't be called the same as the file if it's not the default export, but I'm not sure where I got that from. After removing the underscore ESLint had nothing to say about this